front/app/components/Form/contexts.ts

```diff
@@ -1,14 +1,15 @@
 import { createContext } from 'react';
-import { CLErrors, Message } from 'typings';
+import { CLErrors } from 'typings';
+import { ApiErrorGetter } from '.';
 
 export const APIErrorsContext = createContext<CLErrors | undefined>(undefined);
 
-export type ApiErrorGetter = (
-  errorKey: string,
-  fieldName: string
-) => Message | undefined;
-
 export const FormContext = createContext<{
```

Comment:
This contexts are wrapped around the form and passes down data to the custom components through jsonforms Form component.